components/eamxx/src/share/field/field_alloc_prop.cpp

```diff
@@ -88,7 +88,7 @@ FieldAllocProp FieldAllocProp::subview(const int idim,
   EKAT_REQUIRE_MSG(index_beg < index_end,
                    "Error! Slice indices are invalid (non-increasing).\n");
   EKAT_REQUIRE_MSG(
-      index_beg >= 0 && index_end < m_layout.dim(idim),
+      index_beg >= 0 && index_end <= m_layout.dim(idim),
```

Comment:
Same as comment above about Kokkos bounds

Reply:
I think this is correct since `index_end` can go up to one past the last possible index, which is equal to the dim.